.github/workflows/infection.yml

```
@@ -8,20 +8,15 @@ jobs:
   infection:
     name: Infection
     runs-on: ubuntu-latest
+    env:
+      COMPOSE_PROJECT_NAME: user-service-infection
+      COMPOSE_FILE: docker-compose.yml:docker-compose.memory-tests.yml
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)